deployment/streaming/config-deployment.mk.template

```diff
@@ -9,15 +9,20 @@ export IAM_ACCOUNT :=
 ################################################################################
 #                                                 Streaming resource definitions
 ################################################################################
-export CLUSTER_NAME := osm-stat-stream-cluster
 export STREAMING_INSTANCE_TYPE := m4.xlarge
 export ECR_IMAGE :=
 export AWS_LOG_GROUP := streaming-stats-updater
 export ECS_SUBNET := ${SUBNET}
 export ECS_SECURITY_GROUP :=
 
+export CLUSTER_NAME_DEPLOYMENT :=
```

Comment:
What are these two vars used for? They're not elsewhere in the diff -- can they be removed?